packages/backend/src/services/people/index.ts

```diff
@@ -9,4 +10,25 @@ const findPerson: FindPerson = (id) =>
 
 const getPeopleCount: GetPeopleCount = () => prisma.person.count();
 
-export { findPeople, findPerson, getPeopleCount };
+const addPerson: AddPerson = (person) => {
+  if (person.mass < 0 || person.height <= 0) {
```

Comment:
myślę, że "weight" jest lepszym słowem niż "mass"

Reply:
as we've discussed